src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObj = sourceString
+    .split(';')
+    .map((style) => style.split(':'))
+    .filter((style) => style.length > 1)
+    .reduce((obj, [key, value]) => {
+      obj[key.trim()] = value.trim();
```

Comment:
All good, but try to create a constants, chaining is not a good idea for good readability